src/sentry/event_manager.py

```diff
@@ -2455,6 +2459,52 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects: Projec
                 EventPerformanceProblem(event, performance_problems_by_hash[problem_hash]).save()
 
 
+@metrics.wraps("save_event.send_occurrence_to_platform")
+def _send_occurrence_to_platform(jobs: Sequence[Job], projects: ProjectsMapping) -> None:
+    for job in jobs:
+        event = job["event"]
+        project = event.project
+        event_id = event.event_id
+
+        use_platform = options.get("performance.issues.send_to_issues_platform", False)
+        if use_platform:
+            per_project_option = project.get_option(
+                "sentry:performance_issue_send_to_issues_platform", False
+            )
+            if per_project_option:
```

Comment:
Question about this. How long are we planning to dual-write here by project? I ask because the backfill script I'm working on has some baked in assumptions that we'll be fully dual-writing for all orgs and projects on a certain day.

Is the plan to fully cut-over performance issues to issue platform something like this:
1. Merge this PR to master.
2. Test out dual-writing for sentry org 
3. Remove this check so we can fully dual-write for all org-projects (this would be the date I need for the backfill script to avoid ingesting dupes)
4. Run backfill script
5. Cut-over search queries and stats to query on the issue platform dataset instead of transactions table